compiler_opt/distributed/worker.py

```diff
@@ -28,26 +29,14 @@ def is_priority_method(cls, method_name: str) -> bool:
 
 T = TypeVar('T')
 
-
-# Dask's Futures are limited. This captures that.
-class WorkerFuture(Generic[T], metaclass=abc.ABCMeta):
-
-  @abc.abstractmethod
-  def result(self) -> T:
-    raise NotImplementedError()
-
-  @abc.abstractmethod
-  def done(self) -> bool:
-    raise NotImplementedError()
+WorkerFuture = concurrent.futures.Future
```

Comment:
This was needed to resolve pytype errors, since the scheduler was typed to accept any `concurrent.futures.Future`

Reply:
(and it turns out this doesn't work on 3.8, as I've discovered via github spamming emails, so don't merge this yet)

Reply:
You may want to type it as WorkerFuture, though - see the comment in the 'before' code about Dask Futures.